### PR TITLE
tests: bluetooth: extend bluetooth tag for tests

### PR DESCRIPTION
**Pull request by nordic-piks**
Extend list of paths used to check if bluetooth tests/samples should be executed.

### CODE REVIEW

**Comment by nordic-piks** [comment]
The idea behind this change is to allow automatic detection of PR not related to given code area.
In this case, there is defined tag (e.g. bluetooth) and files/folder with source code related to that area.
Those list of files/folders is then compared to files changed in PR. If particular PR does not change them,
tests related to specified tag should not be executed - will be excluded.

**Comment by nordic-piks** [comment]
Updated after evaluation of:
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml#L192-L213
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/.github/test-spec.yml#L33-L43
https://github.com/nrfconnect/sdk-zephyr/blob/main/.github/test-spec.yml#L57-L92

Please note that the content is similar and syntax is different - the root path is NCS, all dependencies - from west modules - are stated at one place, tests and samples should not be included here, they are handled separately (but they will be build if changed)

**Review by MarekPieta — APPROVED** [review]

**Comment by nordic-piks** [comment]
DNM until NCS 2.7

File: tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml
Comment (kapi-no) [inline tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml:1]: Well, we can drop the `bluetooth` tag from this test project. It is in the Bluetooth category as part of the Fast Pair module but the crypto part does not use the Bluetooth subsystem in any capacity.

File: tests/subsys/bluetooth/fast_pair/storage/account_key_storage/testcase.yaml
Comment (kapi-no) [inline tests/subsys/bluetooth/fast_pair/storage/account_key_storage/testcase.yaml:1]: Well, we can drop the `bluetooth` tag from this test project. It is in the Bluetooth category as part of the Fast Pair module but the storage part does not use the Bluetooth subsystem in any capacity.

File: tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml
Comment (kapi-no) [inline tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml:1]: Well, we can drop the `bluetooth` tag from this test project. It is in the Bluetooth category as part of the Fast Pair module but the storage part does not use the Bluetooth subsystem in any capacity.

**Review by kapi-no — COMMENTED** [review]
You can drop the Bluetooth tag from the crypto and storage test projects in the Fast Pair category

CC: @alstrzebonski @MarekPieta

File: tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml
Comment (nordic-piks) [reply tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml:1]: Thanks, done.

File: tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml
Comment (nordic-piks) [reply tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml:1]: Thanks, done.

File: tests/subsys/bluetooth/fast_pair/storage/account_key_storage/testcase.yaml
Comment (nordic-piks) [reply tests/subsys/bluetooth/fast_pair/storage/account_key_storage/testcase.yaml:1]: Thanks, done.

**Comment by nordic-piks** [comment]
> You can drop the Bluetooth tag from the crypto and storage test projects in the Fast Pair category
> 
> CC: @alstrzebonski @MarekPieta

Thanks, done.

**Review by kapi-no — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by koffes — APPROVED** [review]

**Comment by nordic-piks** [comment]
Just rebased after NCS 2.7

**Pull request merged**